src/pudl_rmi/connect_ferc1_to_eia.py

```diff
@@ -1245,9 +1307,14 @@ def check_match_consistentcy(connects_ferc1_eia, train_df, match_type="all"):
     consistency_one_cap_ferc = 0.9
     mask = connects_ferc1_eia.record_id_eia.notnull()
 
+    if five_year_test:
+        consistency_one_cap_ferc = 0.85
+
     if match_type == "overrides":
         consistency = 0.39
         consistency_one_cap_ferc = 0.83
+        if five_year_test:
+            consistency_one_cap_ferc = 0.75
```

Comment:
if this is the main need for this `five_year_test` flag.... I'd almost be inclined to either:
a) set the 5-year consistency needs as the baseline
b) set these lower consistency requirements if the input `connects_ferc1_eia` has 5 years or less of data in it.

Reply:
Since the consistency didn't change that much, I set the 5 year consistency as the baseline (option a)